composer.json

```
@@ -20,10 +20,10 @@
   "require-dev": {
     "roave/security-advisories": "dev-latest",
     "phpunit/phpunit": "^9.6.7",
-    "vimeo/psalm": "^5.26",
+    "vimeo/psalm": "6.0",
     "friendsofphp/php-cs-fixer": "^3.91",
     "fakerphp/faker": "^1.21",
-    "psalm/plugin-phpunit": "^0.19.0"
+    "psalm/plugin-phpunit": "^0.19.2"
```

The latest available version is [`0.19.5`](https://packagist.org/packages/psalm/plugin-phpunit#0.19.5).

```suggestion
    "psalm/plugin-phpunit": "^0.19"
```

Or 

```suggestion
    "psalm/plugin-phpunit": "^0.19.5"
```

---

but when I update the version the psalm checks fails because of the weak points like 
checking extending of `ClientFactory` and some others errors

---

Psalm is another story, we need to use the correct versions first, then we think about Psalm. For Psalm, we can either fix the issues or just update the [baseline](https://psalm.dev/docs/running_psalm/dealing_with_code_issues/#using-a-baseline-file).

https://github.com/imdhemy/google-play-billing/blob/06eefa673493f9144a62f744d13c48027dca5482/psalm-baseline.xml